### PR TITLE
Implement rds ls

### PR DESCRIPTION
**Pull request by chrishenry**

### CODE REVIEW

File: jungle/rds.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    """return formatted string per instance"""
+    out = []
+    line_format = '{0}\t{1}\t{2}\t{3}'
+    name_len = _get_max_name_len(instances) + 3
+    if flag:
+        line_format = '{0:<' + str(name_len+5) + '}{1:<16}{2:<65}{3:<16}'
+
+    for i in instances:
+        tag_name = i['DBInstanceIdentifier']
+        endpoint = "%s:%s" % (i['Endpoint']['Address'], i['Endpoint']['Port'])
```
Comment (achiku) [inline jungle/rds.py]: Could you use `format` for string format instead of `%`? I just want to make it consistent with other code.

**Comment by achiku** [comment]
@chrishenry I'll merge this after changing `%` to `format`!

**Comment by chrishenry** [comment]
@achiku all set!

**Comment by achiku** [comment]
@chrishenry Thank you for the great work!

**Pull request merged**